common/src/main/java/whocraft/tardis_refined/common/tardis/manager/FlightDanceManager.java

```diff
@@ -33,8 +33,16 @@ public boolean isDancing() {
     }
 
     private List<ControlEntity> getNonCriticalControls(GlobalConsoleBlockEntity controllerConsole) {
-        var controls = controllerConsole.getControlEnttityList();
-        return controls.stream().filter(x -> !(x.controlSpecification().control().equals(TRControlRegistry.THROTTLE.get())) || !(x.controlSpecification().control().equals(TRControlRegistry.HANDBRAKE.get())) || !(x.controlSpecification().control().equals(TRControlRegistry.MONITOR.get()))).toList();
+        var allControlsOnConsole = controllerConsole.getControlEntityList();
+        List<ControlEntity> updatedList = new ArrayList<>(allControlsOnConsole); //Copy over all entries to a new which we can manipulate
+        for(ControlEntity controlEntity : allControlsOnConsole){ //Out of all controls in the original control list, remove any from our copy
+            if(controlEntity.controlSpecification().control() == TRControlRegistry.THROTTLE.get()
```

Comment:
We could perhaps have a boolean on controls for this? If addons decided to add controls they would need to mixin this and we should avoid that if we can

Reply:
Yep agreed, we can add a boolean field on the Control class. Would it be only used for exclusion from the Flight Dance?

Reply:
Added, kept it generically named.

Reply:
@50ap5ud5 this is where I meant to say the tag thing